processYear.ts

```diff
@@ -334,19 +326,33 @@ const selectClusters = async (
           // currently distance is the osrm generated distance between each landing site and the facility location
           const route: any = await getRouteDistanceAndDuration(osrm, routeOptions);
           // number of trips is how many truckloads it takes to transport biomass
-          const numberOfTripsForTransportation = clusterFeedstock / TONS_PER_TRUCK;
+          const FULL_TRUCK_PAYLOAD = 25; // FRCS assumption (in green tons)
+          const numberOfTripsForTransportation = Math.ceil(clusterFeedstock / FULL_TRUCK_PAYLOAD);
           // multiply the osrm road distance by number of trips, transportation eq doubles it for round trip
-          let distance = route.distance;
-          distance = distance / 1000; // m to km
+          const distance = route.distance / 1000; // m to km
           const duration = route.duration / 3600; // seconds to hours
-          const transportationCostPerGT = getTransportationCost(
-            distance,
-            duration,
-            params.dieselFuelPrice
-          );
-          const transportationCostTotal = transportationCostPerGT * clusterFeedstock;
-          const costPerKM =
-            transportationCostTotal / (distance * 2 * numberOfTripsForTransportation);
+          let transportationCostFullPayloadPerGT = 0;
+          if (clusterFeedstock >= FULL_TRUCK_PAYLOAD) {
+            transportationCostFullPayloadPerGT = getTransportationCost(
+              distance,
+              duration,
+              params.dieselFuelPrice,
+              FULL_TRUCK_PAYLOAD
+            );
+          }
+          let transportationCostPartialPayloadPerGT = 0;
+          const partialPayload = clusterFeedstock % FULL_TRUCK_PAYLOAD;
```

Comment:
Might be reading this wrong, but what if feedstock is more than double a truck payload?  won't the mod % just throw away any excess?

Reply:
For example, if `clusterFeedstock` = 55 and `FULL_TRUCK_PAYLOAD` = 25, it will require three round trips to transport the biomass from the cluster to a biopower facility. The first two trips are with a full payload of 25 green tons whereas the last trip is with a partial payload of 5 green tons. mod % gives us the partial payload.

Reply:
oh it's perGT, that's what I was missing.  Got it.